frontend/defaultnations.js

```diff
@@ -0,0 +1,165 @@
+let nations = {
```

Comment:
Just a suggestion, would it be better to make the content portion of this file a JSON text file and assign nations by reading it?  That way the content would be portable to other languages (or just to the backend if needed)

Reply:
Yes, prefer to skip that in this CL though as I want to fold it into the map data itself. (See below.)